.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -378,14 +378,13 @@ jobs:
           git checkout main
           git config --global user.email "${{ secrets.EMAIL }}"
           git config --global user.name  "${{ secrets.USER_NAME }}"
+          git config pull.rebase false  # merge (the default strategy)
```

Comment:
Why does rebase not work correctly?

Reply:
We need to add it according to the error: https://github.com/redhat-performance/benchmark-runner/actions/runs/6169715109/job/16746831451

Reply:
That looks like we simply weren't specifying any strategy at all, and we need to specify something.  Why would rebase or fast-forward not work correctly?

Reply:
No because we need to merge

Reply:
So what exactly needs to be merged (as opposed to rebased)?

Reply:
`hint:   git config pull.rebase false  # merge`

Reply:
It only says that you have to use one of the three strategies.

For that matter, why couldn't you do

```
git pull main
git add "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf/dashboard.json
git commit -m "Update grafana json file"
git push main
```
rather than having to do any kind of merge?

Reply:
Because we are merging, I remembered I got the same error with bump version.

Reply:
Merging what?  It looks like you're just updating dashboard.json, so you should be able to do

```
git checkout main
git pull
git add "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf/dashboard.json
git commit -m "Update grafana json file"
git push
```

Reply:
@RobertKrawitz, 
It will not work, we have already tested it like this and it failed

Reply:
That's what I'm trying to understand.  How did it fail?  It looks to me like you're not merging the file, you're just updating it.

Reply:
@RobertKrawitz We are merging the json file into main.
This is the only option to make it works!

Reply:
I think I see, the push is to the actual repository, so you can't rebase it.  OK, approving.